src/components/atoms/VisusalisationsSelector.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { SelectField } from 'evergreen-ui';
+
+type Props = {
+    label: string;
+    options: Record<string, string>;
```

Comment:
Question:
Is this the same as typing ```options: {x: string, y: string}```?

Reply:
No, not quite. 

`Record<string, string>` is an object where both the keys and the values are strings. Your example would conform to this type, but it would be a subset. 

An alternative typing would be (IIRC, typing from mobile):

`{ [key: string]: string }`

Reply:
Record is very useful when you know the type of the object keys and values, but not the exact key names.

Reply:
Ah ok, looks smart atleast!